### PR TITLE
Metainfo fixes

### PR DESCRIPTION
**Pull request by cbm755**
I don't think the authors would like this work described as "commercial".

IANAL and I don't know the details but I assume the potential issues here involve trademarks, on for example the character names and perhaps likenesses.

### CODE REVIEW

**Comment by cbm755** [comment]
```diff
 <component type="desktop-application">
   <id>org.srb2.SRB2Kart</id>
+  <launchable type="desktop-id">org.srb2.SRB2Kart.desktop</launchable>
   <metadata_license>FSFAP</metadata_license>
```

I had this in: I thought I read somewhere its needed if `<id>` doesn't end in `.desktop` or something like that.

**Comment by cbm755** [comment]
I (previously) did OARS.  Here's the diff:
```dif
   <content_rating type="oars-1.1">
     <content_attribute id="violence-cartoon">moderate</content_attribute>
-    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="language-humor">mild</content_attribute>
+    <content_attribute id="social-chat">mild</content_attribute>
+    <content_attribute id="social-info">mild</content_attribute>
   </content_rating>
```
How should we merge those?

**Comment by ghisvail** [comment]
> I had this in: I thought I read somewhere its needed if \<id\> doesn't end in .desktop or something like that.

The launchable attribute is already [present](https://github.com/ghisvail/flathub/blob/adf36309ff17e01a23fa65afa9c10afa810f3220/org.srb2.SRB2Kart.metainfo.xml#L22).

**Comment by ghisvail** [comment]
> How should we merge those?

Just make a separate PR and I'll merge yours in. These were blindly copied from SRB2 initially.

File: org.srb2.SRB2Kart.metainfo.xml [hunk 1]
```diff
@@ -32,9 +28,11 @@
   </provides>
   <releases>
     <release version="1.2" date="2020-05-12" />
+    <release version="1.1" date="2019-05-28" />
```
Comment (ghisvail) [inline org.srb2.SRB2Kart.metainfo.xml]: We don't need past releases atm.

File: org.srb2.SRB2Kart.metainfo.xml [hunk 2]
```diff
@@ -14,10 +14,6 @@
       and a Battle mode where players fight each other using the items
       available in the game.
     </p>
-    <p>
```
Comment (ghisvail) [inline org.srb2.SRB2Kart.metainfo.xml:17]: This notice was added as a result of the peer review by other Flathub maintainers. The only modification that I agree with is to replace `commercial` with `proprietary`.

**Review by ghisvail — CHANGES_REQUESTED** [review]

**Comment by ghisvail** [comment]
> IANAL and I don't know the details but I assume the potential issues here involve trademarks, on for example the character names and perhaps likenesses.

Some assets are derived from commercial / proprietary trademarks. And the wording regarding the license and redistribution of said trademarks is quite careful [upstream](https://github.com/STJr/Kart-Public/blob/master/assets/README.txt), for good reasons. We should be doing the same.

File: org.srb2.SRB2Kart.metainfo.xml (hunk 2 above)
Comment (ghisvail) [reply org.srb2.SRB2Kart.metainfo.xml:17]: We can remove it now that we provide the assets through extra-data :+1:

Comment (cbm755) [reply org.srb2.SRB2Kart.metainfo.xml:17]: yes sorry that was my intention with this MR (after extra-data)

File: org.srb2.SRB2Kart.metainfo.xml (hunk 1 above)
Comment (cbm755) [reply org.srb2.SRB2Kart.metainfo.xml]: I can't find it now, but I thought I remembered a blog post (possibly by @hughsie) claiming that Gnome Software does/would give some priority based on whether app had "momentum" (recent releases).  Anyway, its no big deal, I'll revert it.

**Pull request merged**